README.md

```diff
@@ -26,14 +18,35 @@ Conjur provides:
 
 * try Conjur: [Start Here](https://try.conjur.org)
 * [support](https://try.conjur.org/support.html)
-* [API Documentation][api-doc]
+* [API Documentation][api]
+
+[![Join Conjur Slack](https://img.shields.io/badge/slack-open-e01563.svg)][slack]
```

Comment:
This whole section should go right under the README title, it's below the fold in Links. Also the header doesn't match the repo name. It should, for consistency - like our other repos.

```
# conjur

badges here

rest of the content
```

example: https://github.com/kubernetes/helm - they've only got 2 but that's where they should go.

Reply:
The url for the doc site should be the URL in the GitHub repo, I'll make sure it's set up this way - so we don't need to link to it from the top of the README either. It's fine below the fold.

Reply:
Done and done